src/bin.ts

```diff
@@ -1,52 +1,33 @@
 #!/usr/bin/env node
 
-import { argv } from 'yargs';
 import path from 'path';
+import yargs from 'yargs';
 import runVrt from './index';
 
-function createRgbArrayFromString(color: string): undefined | [number, number, number] {
-    const rgb = color
-        .replace(/[[\]]/g, '')
-        .split(',')
-        .map((item) => parseInt(item))
-        .filter((item) => !isNaN(item));
+const { cwd, output, teamcity, ...rest } = yargs(process.argv.slice(2))
```

Comment:
why `rest` - not options? mentioned before - this renaming is not necessary. (2)